### ISSUE TITLE
Enabler: FUTURE: Update font references in nvcg.css

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/clinical-trials-search-print#39]
# Load Montserrat fonts from the same location as the rest of Cancer.Gov

Currently, nvcg.css references the Montserrat front files in `/PublishedContent`, this is OK for now as those files are in Akamai NetStorage, but we likely want to use everything from the same location.

nvcg.css is a built artifact, from the wcms-front-end project. This issue is dependent on adding a CSS build system.

> **ESTIMATE** TBD

**Comment by welshja** [comment]
@blairlearn  - is this still valid considering the other changes you made to CTS print to rebuild it?

**Comment by blairlearn** [comment]
Yes, still valid.  The move to S3 kept the same outputs with the same CSS and font references.
